src/day_03/binary_diagnostic.py

```diff
@@ -0,0 +1,108 @@
+import argparse
+from typing import List  # Python 3.8 and earlier
+
+
+def find_power_consumption(diagnostics: List[str]) -> int:
+    """For a given set of diagnostics return the Power Consumption of the Submarine
+
+    Parameters:
+      diagnostics (list): Diagnostics report from Submarine
+
+    Returns:
+      power_consumption (int): Power Consumption of the Submarine
+
+    """
+    diagnostic_reading_length = len(diagnostics[0]) - 1
```

Comment:
Only quibble is diagnostic_reading_length  and total_diagnostic_readings  are only used in one place - so why not just define then there?